### ISSUE TITLE
[Bug]: Pages not responsive in smaller devices

### ISSUE THREAD

**Issue by r7projects-shayan** [ashutosh-rath02/git-re#386]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The navbar is getting hidden and overlapped in small and very small devices

The Faqs section is getting hidden in small and very small devices

### Expected behavior

Every section should be responsive in all devices and dimensions

### Add ScreenShots

![Screenshot 2024-07-15 182341](https://github.com/user-attachments/assets/f6ab38cb-618a-4086-b8a5-340c018765cc)
![Screenshot 2024-07-15 182356](https://github.com/user-attachments/assets/14dc8191-ad74-46c8-bb0c-26f120add2d0)
![Screenshot 2024-07-15 182446](https://github.com/user-attachments/assets/3167eff9-5cf1-459f-a968-60f43a19e608)

### On which device are you experiencing this bug?

Chrome, Windows

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

**Comment by ashutosh-rath02** [comment]
Thank you for creating this issue! 🎉 We'll look into it as soon as possible. In the meantime, please make sure to provide all the necessary details and context. If you have any questions or additional information, feel free to add them here. Your contributions are highly appreciated! 😊

You can also check our [CONTRIBUTING.md](https://github.com/ashutosh-rath02/git-re/blob/main/CONTRIBUTING.md) for guidelines on contributing to this project.

**Comment by r7projects-shayan** [comment]
@ashutosh-rath02 Kindly assign me this major issue

**Comment by ashutosh-rath02** [comment]
Go ahead @r7projects-shayan

**Comment by r7projects-shayan** [comment]
@ashutosh-rath02 Okay

**Comment by ashutosh-rath02** [comment]
@r7projects-shayan update?

**Comment by r7projects-shayan** [comment]
Yeah I'm working on it @ashutosh-rath02 . I will submit it soon with the error fixed, I will provide the screenshots so that you can see
